### PR TITLE
✨AWS library: add start instance method

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- adds method to start instances after they were stopped.
- refactor EC2 client to follow same structure as other AWS clients

## Related issue/s
- required by https://github.com/ITISFoundation/osparc-simcore/pull/6143

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/aws-library/src/aws_library/ec2/_client.py [hunk 1]
```diff
@@ -61,13 +62,13 @@ async def close(self) -> None:
         await self.exit_stack.aclose()
 
     async def ping(self) -> bool:
-        try:
```
Comment (pcrespov) [inline packages/aws-library/src/aws_library/ec2/_client.py:64]: MINOR: This is perfectly valid but I think a try-catch here is actually more clear.

File: packages/aws-library/src/aws_library/ec2/_errors.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from pydantic.errors import PydanticErrorMixin
+
+
+class EC2RuntimeError(PydanticErrorMixin, RuntimeError):
```
Comment (pcrespov) [inline packages/aws-library/src/aws_library/ec2/_errors.py:4]: Why not using our `from models_library.errors_classes import OsparcErrorMixin`

File: packages/aws-library/src/aws_library/ec2/__init__.py
Comment (pcrespov) [inline packages/aws-library/src/aws_library/ec2/__init__.py:1]: much cleaner

**Review by pcrespov — APPROVED** [review]
thx!

File: packages/aws-library/src/aws_library/ec2/_errors.py (hunk 2 above)
Comment (sanderegg) [reply packages/aws-library/src/aws_library/ec2/_errors.py:4]: true that

Comment (sanderegg) [reply packages/aws-library/src/aws_library/ec2/_errors.py:4]: will do another PR to change that in all AWS clients

**Review by matusdrobuliak66 — APPROVED** [review]
thanks, looks good :+1:

**Pull request merged**